src/test/java/org/folio/innreach/domain/listener/KafkaCirculationEventListenerApiTest.java

```diff
@@ -437,14 +450,31 @@ void shouldUpdateTransactionWithPatronHoldWhenCancelRequest() {
     var request = event.getData().getNewEntity();
     request.setId(PRE_POPULATED_PATRON_TRANSACTION_REQUEST_ID);
     request.setStatus(CLOSED_CANCELLED);
+    var inventoryItemDTO = createInventoryItemDTO();
+    var item = new Item();
+
+    when(holdingsService.find(any())).thenReturn(any());
+    when(itemStorageClient.getItemById(ITEM_ID)).thenReturn(Optional.of(item));
+    doNothing().when(inventoryClient).updateItem(ITEM_ID, inventoryItemDTO);
 
     listener.handleRequestEvents(asSingleConsumerRecord(CIRC_REQUEST_TOPIC, PRE_POPULATED_PATRON_TRANSACTION_REQUEST_ID, event));
 
     verify(eventProcessor).process(anyList(), any(Consumer.class));
     verify(innReachExternalService, times(1)).postInnReachApi(any(), any());
 
     var updatedTransaction = transactionRepository.fetchOneById(PRE_POPULATED_PATRON_TRANSACTION_ID).orElse(null);
+    var patronTransaction = (TransactionPatronHold) updatedTransaction.getHold();
     assertEquals(BORROWING_SITE_CANCEL, updatedTransaction.getState());
+    assertEquals(null, patronTransaction.getPatronId());
```

Comment:
there is `assertPatronAndItemInfoCleared()` in this test class to verify patron fields are empty